src/qililab/instruments/qblox/qblox_qrm_rf.py

```diff
@@ -87,6 +87,14 @@ def set_parameter(self, parameter: Parameter, value: ParameterValue, channel_id:
         if parameter == Parameter.LO_FREQUENCY:
             parameter = Parameter.OUT0_IN0_LO_FREQ
 
+        if parameter == Parameter.OUT0_ATT:
+            max_att = self.device._get_max_out_att_0()
+            if value > max_att:
+                raise Exception(
```

Comment:
DRY: instead of repeating the same Exception in three places, we could implement a custom one.

In example, `AttenuationLevelException()` or `MaxAttenuationException()`

Reply:
That's a good point (outside of this PR too). We have a module `exceptions` for this reason, but we haven't used it at all. There are plenty of places in our code that we should raise custom domain-specific errors.

Reply:
Yeah, saw that module and I am creating a custom exception there for wrong set parameters values, a bit generic but good enough I think to begin with.

Reply:
I was creating a custom exception but it goes too deep into changes to be distracted now from this quick PR plus calibrating Contralto. We should tackle custom exceptions in a PR for all instruments and parameters.